igvm/commands.py

```diff
@@ -325,11 +325,15 @@ def vm_build(vm_hostname, run_puppet=True, debug_puppet=False, postboot=None,
 
             if rebuild and vm.hypervisor.vm_defined(vm):
                 vm.hypervisor.undefine_vm(vm)
+                is_rebuild = True
```

Comment:
how about `is_rebuild = rebuild and vm.hypervisor.vm_defined(vm)`
and then do the if based on `is_rebuild` directly. saves us the `else`

Reply:
Changed